main.go

```diff
@@ -67,6 +67,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&opt.Destination, "destination", "stdout", "Log Destination: loki, elasticsearch, stdout, file. (default stdout)")
 	rootCmd.PersistentFlags().Int64Var(&opt.TotalLogLines, "totalLogLines", 0, "Total number of log lines per thread (default 0 - infinite)")
 	rootCmd.PersistentFlags().BoolVar(&opt.DisableSecurityCheck, "disable-security-check", false, "Disable security check in HTTPS client. (default false)")
+	rootCmd.PersistentFlags().StringVar(&opt.BearerTokenFile, "bearer-token-file", "", "Adds a bearer token file. (default service account token)")
```

Comment:
I can not see this default enforced somewhere. Does it happen automatically in the Prometheus code somewhere?

Reply:
Do you mean in terms of ensuring that the flag is it's default value or in the code? For instance, [here](https://github.com/ViaQ/cluster-logging-load-client/pull/22/files#diff-ccbd22021186620d367fc75b54306420d0e051545aa5f5d0eaa90139a8bcd154R288) is a check to ensure that only the path is configured.

Reply:
Sorry for not making my point clearer. I meant what enforced the "default value" which according to the help is "service account token". In this context this could be the token of the Kubernetes ServiceAccount this is running under (`/var/run/secrets/kubernetes.io/serviceaccount/token`).

It's also possible that I misunderstand the meaning of the help text.

Reply:
Ohh, I see. No, I think the help text is not well phrased. I'll update it.